frontend/src/app/logs/logs.component.html

```diff
@@ -1,122 +1,205 @@
-              }
-            </oib-multi-select>
-          </div>
+      <div class="row gx-2 py-3">
+        <label for="levels" class="col-auto col-form-label" translate="logs.search.levels"></label>
+        <div class="col-2">
+          <oib-multi-select formControlName="levels" id="levels">
+            @for (level of levels; track level) {
+              <oib-multi-select-option [value]="level" [label]="level | logLevelsEnum"></oib-multi-select-option>
+            }
+          </oib-multi-select>
+        </div>
+        <label for="message-content" class="col-auto col-form-label" translate="logs.search.message-content"></label>
+        <div class="col-7">
+          <input type="text" class="form-control" id="message-content" formControlName="messageContent" />
+        </div>
+
+        <div class="col">
+          <button class="btn btn-primary pull-right" id="search-button" [disabled]="loading">
+            @if (loading) {
+              <span class="fa fa-spin fa-spinner"></span>
+            } @else {
+              <span translate="common.search"></span>
+            }
+          </button>
+        </div>
+      </div>
+    </form>
 
-          <label for="scope-types" class="col-auto col-form-label" translate="logs.search.scope-type"></label>
-          <div class="col-2">
-            <oib-multi-select formControlName="scopeTypes" id="scope-types">
-              @for (scopeType of scopeTypes; track scopeType) {
-                <oib-multi-select-option [value]="scopeType" [label]="scopeType | scopeTypesEnum"></oib-multi-select-option>
-              }
-            </oib-multi-select>
+    @if (logs.totalElements) {
+      <div class="d-flex justify-content-between mb-2">
+        @if (this.levels && this.levels.length > 0) {
+          <oib-legend [legendList]="LEGEND"></oib-legend>
+        }
+        <div [class]="logs.totalPages > 1 ? 'ms-2' : ''">
+          <oib-pagination [page]="logs" [navigate]="true"></oib-pagination>
+        </div>
+      </div>
+    }
+    @if (logs.totalElements !== 0) {
+      <div class="table-container">
+        <table class="table table-sm table-hover oib-table-logs">
+          <thead class="light">
+            <tr>
+              <th style="width: 1%"></th>
+              <th translate="logs.date"></th>
+              <th translate="logs.scope-type" style="width: 8%"></th>
+              <th translate="logs.scope" style="width: 4%"></th>
+              <th translate="logs.message"></th>
+            </tr>
+          </thead>
+          <tbody>
+            @for (log of logs.content; track log) {
+              <tr class="data">
+                <td>
+                  <span style="display: block" [class]="getLevelClass(log.level)"></span>
+                </td>
+                <td class="text-nowrap">{{ log.timestamp | datetime: 'mediumWithSeconds' }}</td>
+                <td>{{ log.scopeType | scopeTypesEnum }}</td>
+                <td>{{ log.scopeName }}</td>
```

Comment:
Display these columns only if scopeType and scopeId is null (otherwise we know on which type and which connector we are)